### PR TITLE
Bump CAPI version and remove design type logic

### PR DESCRIPTION
**Pull request by nicl**
Small version bump, which allows us to remove some logic as it now lives in the client itself.

### CODE REVIEW

**Review by regiskuckaertz — APPROVED** [review]

File: common/app/model/Formats.scala [hunk 1]
```diff
@@ -159,6 +159,7 @@ object PressedContentFormat {
       case JsString("GuardianView") => JsSuccess(com.gu.contentapi.client.utils.GuardianView)
       case JsString("Quiz") => JsSuccess(com.gu.contentapi.client.utils.Quiz)
       case JsString("GuardianLabs") => JsSuccess(com.gu.contentapi.client.utils.GuardianLabs)
+      case JsString("AvertisementFeature") => JsSuccess(com.gu.contentapi.client.utils.AdvertisementFeature)
```
Comment (tjmw) [inline common/app/model/Formats.scala]:
```suggestion
      case JsString("AdvertisementFeature") => JsSuccess(com.gu.contentapi.client.utils.AdvertisementFeature)
```

**Review by tjmw — APPROVED** [review]
:+1: I spotted what looks like a typo, but otherwise LGTM!

**Comment by tjmw** [comment]
👋 Hi @nicl - is there anything I can do to help get this over the line? During an incident on Friday, Regis mentioned some benefits on being on the latest Capi client version.

**Comment by mxdvl** [comment]
Hi there!
Should this PR remain open? It's been around for a while and should probably either ship or be closed.
If it needs to remain open, could you please change the base branch to `main`? See mention above for context.

🧹💨 @nicl @tjmw

**Pull request closed without merging**